packages/react-native-sdk/docusaurus/docs/reactnative/06-advanced/08-screensharing/02-setup/01-react-native.mdx

```diff
@@ -19,7 +19,6 @@ To be able to use the foreground service, the permission must be declared in the
 <uses-permission android:name="android.permission.POST_NOTIFICATIONS" />
 <!-- We declare the permissions to for using foreground service -->
 <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
-<uses-permission android:name="android.permission.FOREGROUND_SERVICE_MICROPHONE" />
```

Comment:
Just a double-check. This isn't needed anymore?

Reply:
yes it isnt